src/apis/index.js

```diff
@@ -36,6 +36,11 @@ export function getChangePassword(payload) {
   });
 }
 
+export function getAdmin() {
+  const url = `${API_URL}/${AUTH_API_PREFIX}/showAdminService.json}`;
```

Comment:
Remove `}` at the end of the url. It causes API to fail.

Did it work for you?

Also, the correct demo link is `https://pr-1842-fossasia-susi-web-chat.surge.sh/`. Kindly update it.

Reply:
@akshatnitd made the changes